ruby/ruby_trunk/ruby_benchmarks/runner

```diff
@@ -54,3 +54,14 @@ if [ "$OPTCARROT_BENCHMARK" = true ]; then
 else
   echo "Skipping optcarrot benchmarks"
 fi
+
+if [ "$LIQUID_BENCHMARK" = true ]; then
```

Comment:
@kirs I played around with this, and it started getting weird. 

If we're trying to share a bash file, that would mean that bash file goes in the parent directory of `ruby_releases` and `ruby_trunk`. However, this means that the Dockerfiles inside `ruby_releases` and `ruby_trunk` can't add the shared file to the image since it's out of the build context.

We could move the build scripts outside of the `ruby_releases` and `ruby_trunk` directories, but then I'm not sure how this would affect things server-side.